__tests__/fixtures.ts

```diff
@@ -2,14 +2,29 @@ import fs from 'fs';
 import path from 'path';
 
 import { Account, ProviderInterface, RpcProvider, SequencerProvider, json } from '../src';
-import { CompiledContract, waitForTransactionOptions } from '../src/types';
+import {
+  CompiledContract,
+  CompiledSiera,
+  CompiledSieraCasm,
+  waitForTransactionOptions,
+} from '../src/types';
 import { toHex } from '../src/utils/num';
 
 const readContract = (name: string): CompiledContract =>
```

Comment:
```suggestion
const readContract = <T = CompiledContract>(name: string, extension: string = 'json'): T =>
  json.parse(
    fs.readFileSync(path.resolve(__dirname, `../__mocks__/${name}.${extension}`)).toString('ascii')
  )

...

export const compiledHelloSiera = readContract<CompiledSiera>('cairo/helloSiera/hello')
export const compiledHelloSieraCasm = readContract<CompiledSieraCasm>('cairo/helloSiera/hello', 'casm')
```

Reply:
I like the generic approach (for permanent solutions) but this and the hash file will be cleaned up soon as we progress to remote compile and integrated hashes in compilation, so it is temporary and easy to remove.